README.rst

```diff
@@ -25,6 +25,39 @@ The latest version of the schema may always be found in ``schema/latest.txt``.
 
 .. _Confluent compatibility model: https://docs.confluent.io/current/schema-registry/docs/avro.html#forward-compatibility
 
+Adding a new schema
+-------------------
+
+Steps to update the alert schema (for example, when the APDB schema is updated):
+
+* ``setup -r .`` in this package's root.
+* Checkout the ticket branch for your schema changes.
+* Update the default ``schema_root`` kwarg in ``python/lsst/alert/packet/schemaRegistry.py:from_filesystem()`` to your new schema version number.
+* Add a new schema in ``python/lsst/alert/packet/schema/``. Subsequent instructions assume you are in that directory.
+
+   * Recursively copy the latest schema directory to the next number (e.g. ``cp -rp 4/ 5/`` to add schema version 5 based on version 4). Subsequent directions assume you are in this new directory.
```

Comment:
This is a little too general: non-breaking schema (e.g., adding a column) changes will generate minor versions (e.g., `4/0` → `4/1`).  See https://dmtn-093.lsst.io/#management-and-evolution for discussion.

Reply:
I've rephrased this section and added a new bullet at the top. See the latest commit.